libs/actions-runner-controller/config.jsonnet

```diff
@@ -0,0 +1,23 @@
+local config = import 'jsonnet/config.jsonnet';
+
+local versions = ['0.20.1'];
```

Comment:
In every other library, we don't put the patch number in the `output` value (line 10). For consistency, I think it'd be better to omit it

Reply:
What happens when the CRDs are updated in a patch release on the upstream repo in that scenario?

Reply:
On a patch, I would hope that the upstream repository would not introduce changes to CRDs that would result in breaking changes in libraries here. I think we could just bump the patch version here while keeping the same export path

Reply:
For this project in particular the diff between 0.20.0 and 0.20.3 is quite large for example
```
Hamish-MBP ~/g/actions-runner-controller:(HEAD detached at v0.20.0)> git diff --stat v0.20.3 charts/actions-runner-controller/crds
 charts/actions-runner-controller/crds/actions.summerwind.dev_horizontalrunnerautoscalers.yaml |   489 +++++-----
 charts/actions-runner-controller/crds/actions.summerwind.dev_runnerdeployments.yaml           | 12046 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++--------------------------------------------------------------------------------------
 charts/actions-runner-controller/crds/actions.summerwind.dev_runnerreplicasets.yaml           | 12026 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++--------------------------------------------------------------------------------------
 charts/actions-runner-controller/crds/actions.summerwind.dev_runners.yaml                     | 11473 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++--------------------------------------------------------------------------------------
 charts/actions-runner-controller/crds/actions.summerwind.dev_runnersets.yaml                  | 10649 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++---------------------------------------------------------------------------
 5 files changed, 29203 insertions(+), 17480 deletions(-)
 
```

Reply:
> On a patch, I would hope that the upstream repository would not introduce changes to CRDs that would result in breaking changes in libraries here. I think we could just bump the patch version here while keeping the same export path

I'd hope so too!   
On the other hand we don't control what upstream repos do but we can control what we do?

Guess we can cross that bridge when we come to it :)

Reply:
That's a good example 😅 . I'm ok with leaving the patch here

Reply:
I think they added a linter and it's all whitespace changes actually, hard to tell because the diff is so huge but...